### PR TITLE
fix(api,cli,doc): add ui url + update doc

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: sdk/v2_workflow.go [hunk 1]
```diff
@@ -496,3 +496,8 @@ type V2WorkflowRunManualRequest struct {
 	WorkflowBranch string `json:"workflow_branch,omitempty"`
 	WorkflowTag    string `json:"workflow_tag,omitempty"`
 }
+
+type V2WorkflowRunManualResponse struct {
+	HookEventUUID string `json:"string"`
```
Comment (yesnault) [inline sdk/v2_workflow.go:501]:
```suggestion
	HookEventUUID string `json:"hook_event_uuid"`
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20454.0 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report build-cds#20457.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20457.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**